h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -66,6 +66,28 @@ public boolean haveMojo() {
     return true;
   }
 
+  @Override
+  public void init(boolean expensive) {
+    super.init(expensive);
+    
+    if (_parms._metalearner_fold_column != null && _parms._train != null) {
+      final Frame trainingFrame = _parms._train.get();
+      if(trainingFrame.vec(_parms._metalearner_fold_column) == null) {
+        throw new IllegalArgumentException(String.format("Specified fold column '%s' not found in the training data frame: '%s'. Available column names are: %s",
+                _parms._metalearner_fold_column, trainingFrame._key, Arrays.toString(trainingFrame.names())));
+      }
+    }
+
+    if (_parms._metalearner_fold_column != null && _parms._valid != null) {
+      final Frame validationFrame = _parms._valid.get();
+      if(validationFrame.vec(_parms._metalearner_fold_column) == null) {
+        throw new IllegalArgumentException(String.format("Specified fold column '%s' not found in the validation data frame: '%s'. Available column names are: %s",
+                _parms._metalearner_fold_column, validationFrame._key, Arrays.toString(validationFrame.names())));
+      }
+    }
+    
+  }
```

Comment:
@Pscheidl nice thanks! Can you please also add similar logic for `_parms._blending`?